From 2421edb8e799dc067e2091452001ac1450ea8345 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Fri, 17 Jan 2020 14:45:39 +0000
Subject: [PATCH] xvmc: add missing dependency on Xv

libXvMC does not directly pull in libXv since dd9ae03 (1.0.12 onwards).
Add the missing Xv depenedency as the driver does need libXv.

Closes #180
---
 xvmc/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/xvmc/meson.build b/xvmc/meson.build
index 0ecc51e57..eeba2b669 100644
--- a/xvmc/meson.build
+++ b/xvmc/meson.build
@@ -14,6 +14,7 @@ shared_library('IntelXvMC',
 	       dependencies : [
 		 dependency('threads', required : true),
 		 dependency('x11', required : true),
+		 dependency('xv', required : true),
 		 dependency('xvmc', required : true),
 		 dependency('xorg-server', required : true),
 		 dependency('x11-xcb', required : true),
-- 
GitLab

